examples/bin/SparseSparseDrmTimer.mscala

```diff
@@ -0,0 +1,37 @@
+
+
+def timeSparseDRMMMul(m: Int, n: Int, s: Int, para: Int, pctDense: Double = .20): Long = {
+
+ 
+
+  val drmA = drmParallelizeEmpty(m , s, para).mapBlock(){
+       case (keys,block:Matrix) =>
+         val R =  scala.util.Random
+         R.setSeed(1234L)
```

Comment:
If we use hardcoded seeds, we should probably define them as a constant somewhere